state/indexer/sink/psql/schema.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TYPE block_event_type AS ENUM ('begin_block', 'end_block', '');
+CREATE TABLE block_events (
+    block_event_id SERIAL PRIMARY KEY,
```

Comment:
```suggestion
    id SERIAL PRIMARY KEY,
```

Reply:
If I use `id` to the table's Primary key, 
I will get an error `pq: column reference "id" is ambiguous` when doing the query.

Reply:
I don't follow. What data is being verified? Using `id` is the idiomatic way to name this column. Do you have an example?

Reply:
The column should be `id`. Can you type the exact SQL query here please? The issue is probably with your query.

Reply:
I just found I can use the table name/alias in the query. Let me try it. Thanks!